engine/table/src/main/java/io/deephaven/engine/table/impl/util/BarrageMessage.java

```diff
@@ -46,6 +46,7 @@ public static class AddColumnData {
     public long firstSeq = -1;
     public long lastSeq = -1;
     public long step = -1;
+    public long tableSize = -1;
```

Comment:
I think we should be consistent about setting this. You should audit BarrageMessage creation (e.g. wherever we set `rowsAdded`).